app/controllers/users_controller.rb

```diff
@@ -10,6 +10,23 @@ def create
 
   def show
     render file: "/public/404" unless current_user
+    @user = current_user
```

Comment:
`current_user` is available as a helper method from `application_controller`
You can call `current_user` from the view without having to pass an instance variable in this method.

```erb
<h1>Profile Page</h1>
<h2>Name: <%= current_user.name %></h2>
<h2>Street Address: <%= current_user.address %></h2>
<h2>City: <%= current_user.city %></h2>
<h2>State: <%= current_user.state %></h2>
<h2>ZIP Code: <%= current_user.zip %></h2>
<h2>Email Address: <%= current_user.email %></h2>
<%= link_to "Edit My Profile", "/profile/edit" %>
<%= link_to "Change My Password", "/profile/password/edit" %>
```

(I ran your spec against this and it still passes 😄 )